lib/index.js

```diff
@@ -5,6 +5,13 @@ import AvatarTitle from './AvatarTitle';
 import Button from './Button';
 
 import IconArrow from './IconArrow';
+import IconClear from './IconClear';
+import IconNotification from './IconNotification';
+import IconRequired from './IconRequired';
+import IconSearch from './IconSearch';
+
+import TextField from './TextField';
+import TextArea from './TextArea';
```

Comment:
Again, using extended exports:

```js
export { default as Button } from './Button'
export { default as IconArrow } from './IconArrow'
```

etc.

Reply:
This is explicitly forbidden by the Airbnb JS code style, see https://github.com/airbnb/javascript#modules--no-export-from-import

If we want to use this style, we should amend our code styleguide to allow it.